### PR TITLE
v2.0

### PR DESCRIPTION
**Pull request by JayGray**

### CODE REVIEW

File: js/urlando.js [hunk 1]
```diff
@@ -115,11 +117,13 @@ class Urlando extends Main {
         const currentViewNr = this.currentView.getIdx();
         const prevViewNr = currentViewNr - 1;
 
+        this.currentView.trigger('inactive');
         this.currentView = this.views[prevViewNr < 0 ? nrOfViews -1 : prevViewNr];
         this.transitionToView();
     }
 
     prepareOptions(options) {
+        console.log(options);
```
Comment (mschaaf) [inline js/urlando.js]: please make a debug out of it

File: js/urlando.js [hunk 2]
```diff
@@ -129,6 +133,7 @@ class Urlando extends Main {
             urls = options.urls;
         }
 
+
```
Comment (mschaaf) [inline js/urlando.js]: one empty lien to much ;-)

File: js/urlando.js [hunk 3]
```diff
@@ -192,6 +199,10 @@ class Urlando extends Main {
 
         this.showOverview();
         setTimeout(() => {
+            const now = Date.now();
+
+            this.timeToNextSwitch = now + duration;
```
Comment (mschaaf) [inline js/urlando.js]: Where is this used?

**Review by mschaaf — CHANGES_REQUESTED** [review]

**Review by mschaaf — CHANGES_REQUESTED** [review]
Please document also new keys and behaviour in the README. Thank you.

**Comment by JayGray** [comment]
New keys already documented.

File: js/urlando.js (hunk 3 above)
Comment (JayGray) [inline js/urlando.js]: Nowhere. Removed.

File: js/urlando.js (hunk 1 above)
Comment (JayGray) [inline js/urlando.js]: Removed.

File: js/urlando.js (hunk 2 above)
Comment (JayGray) [inline js/urlando.js]: Removed.

**Review by mschaaf — APPROVED** [review]

**Pull request merged**